src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -2220,6 +2220,14 @@ public void testImageLabels() throws DockerException,
     final List<Image> preexistingTestImages = sut.listImages(
             ListImagesParam.withLabel("name", "testtesttest"));
     final int numPreexistingTestImages = preexistingTestImages.size();
+    // Make a list of just the image IDs
+    final List<String> preexistingTestImageIds =
+        Lists.transform(preexistingTestImages, new Function<Image, String>() {
```

Comment:
Could just do

    final List<String> preexistingTestImageIds = imagesToShortIds(preexistingTestImages);

if you’re ok with the short (12-hex long) id.